docs/guides/supported-file-formats.mdx

```diff
@@ -10,14 +10,14 @@ Vertex supports the following file formats. We regularly add additional support.
 | ---------------- | ------------------------------------------------ | -------------------------- |
 | 3DXML            | .3DXML                                           | All                        |
 | CATIA            | .CATProduct\*, .CATPart, .model, .CATShape, .cgr | 4-6                        |
-| Creo/Pro-E       | .asm\*, .prt, .neu                               | Pro/Engineer 19.0-Creo 7.0 |
```

Comment:
Do we want to update these in the beta docs too? `versioned_docs/version-beta/guides/importing-data.md`